### PR TITLE
Feature/134678595/facebook like button

### PR DESCRIPTION
**Pull request by andela-uenabulele**
#### What does this PR do?
It adds a facebook-like button with thumbnail pictures of some facebook users that already liked the page
#### Description of Task that was completed?
Added the "facebook like" script from facebook to the index page, and classes for styling. This script will be edited to reflect real data on the new UI landing page.
#### How should this be manually tested?
- Run this app locally or [click here](https://kissa-cfh-staging-pr-47.herokuapp.com/) to visit our Heroku site.
- You will find the facebook like button on the landing page
#### What are the relevant pivotal tracker stories?
#134678595
#### Questions:

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -15,6 +15,14 @@
    top: -65px;
 }
 
+.margin-left-20{
+   margin-left: 20%;
+}
+
+.pad-top-2{
+   padding-top: 2%
```
Comment (andela-jare) [inline public/css/style.css]: Missing semi-colon at the end of line 23

File: public/views/index.html [hunk 2]
```diff
@@ -121,6 +121,10 @@ <h3 id="slogan">What is Cards for Humanity?</h3>
               </span>
         </span>
        </div>
+       <div class="text-center margin-left-20 pad-top-2">
+             <iframe src="https://www.facebook.com/plugins/like.php?href=https%3A%2F%2Fdevelopers.facebook.com%2Fdocs%2Fplugins%2F&width=450&layout=standard&action=like&size=small&show_faces=true&share=true&height=80&appId=236093993111473" width="450" height="80" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true"></iframe>
+
```
Comment (andela-jare) [inline public/views/index.html]: Remove the space

**Comment by andela-tisrael** [comment]
Good job fixing the houndCI issue

**Review by vonvick — DISMISSED** [review]

File: public/css/style.css [hunk 3]
```diff
@@ -15,6 +15,11 @@
    top: -65px;
 }
 
+.facebook-like{
+   margin-left: 20%;
```
Comment (andela-vnwaiwu) [inline public/css/style.css]: good css naming style

File: public/css/style.css [hunk 4]
```diff
@@ -15,6 +15,15 @@
    top: -65px;
 }
 
+.facebook-like-container{
+padding-top: 3%
```
Comment (tomily1) [inline public/css/style.css]: **Nitpick**
semicolon after the css statement

File: public/views/index.html [hunk 5]
```diff
@@ -121,6 +121,16 @@ <h3 id="slogan">What is Cards for Humanity?</h3>
...
         </span>
        </div>
+       <div class="text-center facebook-like-container ">
+        <div class="col-md-4">
+        </div>
+       <div class="col-md-4 text-center facebook-like">
+       <a href="https://www.facebook.com/Cards-For-Humanity-701709783330407/"> Visit Our Facebook Page</a>
+             <iframe src="https://www.facebook.com/plugins/like.php?href=https%3A%2F%2Fwww.facebook.com%2FCards-For-Humanity-701709783330407%2F&width=450&layout=standard&action=like&show_faces=true&share=false&height=80&appId" width="450" height="80" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true"></iframe>
+</div>
+ <div class="col-md-4">
```
Comment (tomily1) [inline public/views/index.html]: **Nitpick**
Irrelevant div tag?

Comment (andela-uenabulele) [reply public/views/index.html]: Thanks, it's actually important to complete the 12column grid implementation in bootstrap.

**Review by oagboola — APPROVED** [review]
LGTM

File: public/css/style.css (hunk 1 above)
Comment (makinwab) [reply public/css/style.css]: 👍

**Comment by andela-uenabulele** [comment]
Please, what does LGTM mean?

**Pull request merged**